routes/products.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database');
+
+router.route('/')
+  .get(function (req, res) {
+    knex.select('id', 'name', 'price', 'inventory').from('products')
```

Comment:
Normally you wouldn't put all the methods on one line, it's easier to read if each method was on its own line.